web/src/app/workplace/taskdetails/general/general-fields.component.ts

```diff
@@ -47,20 +45,10 @@ export class TaskdetailsGeneralFieldsComponent implements OnInit, OnChanges {
     }
   }
 
-  ngOnInit() {
```

Comment:
Why did you move getClassifications from general-fields.component to task-details? I think that task-details doesn´t need to know about classifications.

Reply:
When you move back getClassification to this component, please make a test to check that classification service is called on init.

Reply:
done!